backend/EasySpeak.Core/EasySpeak.Core.BLL/Services/UserService.cs

```diff
@@ -53,5 +58,28 @@ public async Task<UserDto> GetUserAsync()
 
             return _mapper.Map<UserDto>(user);
         }
+
+        public async Task<List<UserShortInfoDto>> GetFilteredUsers(UserFilterDto userFilter)
+        {
+            var users = _context.Users.Include(u => u.Tags);
+
+            IQueryable<User> filteredUsers = users;
+
+            if(userFilter.Language is not null)
+            {
+                var enumLanguage = _mapper.Map<Language>(userFilter.Language);
+                filteredUsers = filteredUsers.Where(u => u.Language == enumLanguage);
+            }
+            if(userFilter.LangLevels is not null && userFilter.LangLevels.Length != 0)
```

Comment:
It's better to change userFilter.LangLevels.Length -> userFilter.LangLevels.Any()